pcsx2/gui/AppMain.cpp

```diff
@@ -563,6 +586,23 @@ void Pcsx2App::LogicalVsync()
 		}
 	}
 
+	if (g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
+		if (EnableFMV == 1) {
+			Console.Warning("FMV on");
+			Do43Switch(1);
+			EnableFMV = 0;
+		}
+
+		if (FMVstarted) {
+			int diff = cpuRegs.cycle - eecount_on_last_vdec;
+			if (diff > 60000000) {
+				Console.Warning("FMV off");
+				Do43Switch(0);
```

Comment:
This console warning might not be useful for the users,  Maybe change it to something so they it only triggers for dev builds and debug builds ? (In case you need it for debugging)